kytos/core/interface.py

```diff
@@ -494,7 +494,8 @@ def __eq__(self, other):
 
     def is_valid(self):
         """Check if TAG is possible for this interface TAG pool."""
-        if self.user_tag:
+        if self.user_tag and \
```

Comment:
@Alopalao this cover the requirements of rejecting unsupported strs, but then it's doing a O(n) lookup TAG `__eq__` comparions with a lock, it won't be a bottleneck since we know this list isn't massive large, but we could do better here with an early return for instance considering that we know based on our requirements that the only supported strs are `"any" or "untagged"` for a given `TAG`, could you adapt this?

Reply:
I was thinking in creating a function that handles only strings on `TAG.value`, If it is a better option is this function better inside UNI or Interface class?

Reply:
That's a good idea @Alopalao, let's go for it, feel free to create a new method on `UNI` something like `_is_reserved_valid_tag(self, value) -> bool`, and then early on `is_valid(self)` you call this new method and if it's reserved valid its returned early ortherwise `return self.interface.is_tag_available(self.user_tag)`

Reply:
You can also include `None` as a "reserved valid tag" to simplify some conditionals

Reply:
Ok